### DISCUSSION TITLE
Triggering a workflow from another workflow?

### DISCUSSION THREAD

**Discussion by joyofdata** [argoproj/argo-workflows#5154] [Q&A]
Hello,

what would be the/a recommended method to trigger a workflow B from a workflow A without introducing a parent workflow chaining A and B.

I did read the manual but as I am quite new to K8s and Argo I didn't recognize the proper solution to my problem. Promising features are the REST API and WebHooks if I understand correctly.

Would this be a use case for Argo Events?

I'd like to differentiate between to scenarios:

1) A always triggers B after it completed
2) A triggers B depending on a condition

Setup-wise I'd like to differentiate between:

1) B is triggered by a configuration in a template
2) B is triggered programmatically by Python/Bash in a container

Thanks

**Comment by ghost** [comment]
I did find this, but curl does not seem adequate for such a basic feature as triggering other workflows:
https://argoproj.github.io/argo-workflows/workflow-submitting-workflow/

Argo Events is another possibility, but is observability of the relationships between workflows lost when this is used?
https://argoproj.github.io/argo-events/

A control-m like ability to chain workflows would be very useful.
